### PR TITLE
[Doc] 添加CTAS文档

### PR DESCRIPTION
**Pull request by stalary**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by hf200012 — DISMISSED** [review]
LGTM

File: docs/en/sql-manual/sql-reference/Data-Definition-Statements/Create/CREATE-TABLE-AS-SELECT.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+```sql
+CREATE TABLE table_name [( column_name_list )]
+opt_engine opt_partition opt_properties KW_AS query_stmt
+ ```
+
+illustrate: 
+
+- Fields of type`decimal`are not currently supported
+- The user needs the`SELECT`permission for the table and the`CREATE`permission for the target library
```
Comment (wangshuo128) [inline docs/en/sql-manual/sql-reference/Data-Definition-Statements/Create/CREATE-TABLE-AS-SELECT.md:47]:
```suggestion
- The user needs the`SELECT`permission for the source table and the`CREATE`permission for the target database
```

File: docs/zh-CN/sql-manual/sql-reference/Data-Definition-Statements/Create/CREATE-TABLE-AS-SELECT.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+```sql
+CREATE TABLE table_name [( column_name_list )]
+opt_engine opt_partition opt_properties KW_AS query_stmt
+ ```
+
+说明: 
+
+- 暂时不支持`decimal`类型的字段
+- 用户需要对应表的`SELECT`权限和目标库的`CREATE`权限
```
Comment (wangshuo128) [inline docs/zh-CN/sql-manual/sql-reference/Data-Definition-Statements/Create/CREATE-TABLE-AS-SELECT.md:47]:
```suggestion
- 用户需要对来源表的`SELECT`权限和目标库的`CREATE`权限
```

**Comment by stalary** [comment]
done. @wangshuo128

**Review by hf200012 — APPROVED** [review]
LGTM

**Pull request merged**